g3doc/user_guide/tutorials/knative.md

```diff
@@ -12,46 +12,56 @@ workloads. This could be a
 cluster on Google Cloud Platform or one you have set up yourself using
 [containerd Quick Start](https://gvisor.dev/docs/user_guide/containerd/quick_start/).
 
-This guide will also assume you have Knative installed using
-[Istio](https://istio.io/) as the network layer. You can follow the
+Follow the
```

Comment:
Incomplete sentence

Reply:
Please see the lines below. Generally markdown doesn't count this as a newline.
See 16-18 before the changes

Reply:
Ah right.